### PR TITLE
Data providers tests for Achainable, Nodereal, Discord, Litentry, and Twitter

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

this pr: 
resolves p-367
resolves p-368
resolves p-369
resolves p-370

Add all vcs request json(If anything is missing, please let me know, and will add it.)
- [x] Achainable
   - Amount Holding
   - Amount
   - Basic
   - Class of year
   - Brc20 holder
 - [x] Noderal
      - Evm amount holding(v2)
      - Bnb domain
      - Nft holder
      - LIT Staking
 - [x] Discord
   - Active Discord ID-Hubber
   - Litentry Discord Member
   - Litentry & SORA Quiz Attendee
   - Litentry & Ordinals User
   - Contest Legend
   - Contest Popularity
   - Contest Participant
 - [x] Twitter
 - [x] Vip
    - Gold
    - Silver
  - [x] Litentry
    - DHub EVM Version Early Bird
 - [x] oneblock
    - oneblock-course-participation
    - oneblock-course-completion
    - oneblock-course-outstanding

Note:
Since [mockDid](https://github.com/litentry/litentry-parachain/pull/2434/files#diff-8fa63acdeaf76089ff6e00da4e4ed84ffb2ec55f4f4cf1bb5e99c1289eb56009R17) is a real address, but uncontrollable, it can lead to uncertainty about whether [expectedCredentialValue](https://github.com/litentry/litentry-parachain/pull/2434/files#diff-8fa63acdeaf76089ff6e00da4e4ed84ffb2ec55f4f4cf1bb5e99c1289eb56009R19) is `true` or `false`. If we want to test, I believe we need to find the address yourself, update it, and then run GHA to check the results.

### CODE REVIEW

**Comment by outofxxx** [comment]
~~`MATIC` is currently  not supported~~

Confirmed from @higherordertech , it has been supported in V2.
```rust
	#[codec(index = 24)]
	TokenHoldingAmount(Web3TokenType),
```

File: tee-worker/ts-tests/integration-tests/common/credential-json/litentry.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            "id": "A20",
+            "payload": []
+        },
+        "dataProvider": "litentryIndexer",
+        "network": "litentry",
+        "mockDid": "litentry:substrate:0xa20d2c66bd88271ce78b2d8b7367c025e108944fda26a5a4e95b5efcc3f26b45",
+        "mockWeb3Network": "litentry,polkadot",
+        "expectedCredentialValue": false
+    }
+]
```
Comment (Traf333) [inline tee-worker/ts-tests/integration-tests/common/credential-json/litentry.json:16]: is there any option that we have a successful result and unsuccessful? Like early bird user and non EB user

**Review by Traf333 — APPROVED** [review]
Looks good to me. ❤️

File: tee-worker/ts-tests/integration-tests/common/credential-json/litentry.json (hunk 1 above)
Comment (0xverin) [reply tee-worker/ts-tests/integration-tests/common/credential-json/litentry.json:16]: Of course, It depends on the configuration of [mockDid](https://github.com/litentry/litentry-parachain/pull/2434/files#diff-2f5b9536b27ee7421a35858e8154ebec68e6cde74df92a3788b680f952ce7ac8R12); that's the purpose of this PR. You can try it locally.

Comment (kziemianek) [reply tee-worker/ts-tests/integration-tests/common/credential-json/litentry.json:16]: You can use [this address](https://github.com/litentry/litentry-parachain/blob/4d7fdcbafa78776cac309db128fe4a2a83927c69/tee-worker/litentry/core/mock-server/src/litentry_archive.rs#L29) if you want to have a credential with `true` value

**Pull request merged**